web/parts/Dockerfile-node7

```diff
@@ -0,0 +1,11 @@
+arg node_release=node_7.x
```

Comment:
Yes! I was thinking about that, but I was reaching the point where the Dockerfile-generating code gets unnecessarily complex, and chose to duplicate things a little instead of being infinitely (and unmaintainably) clever.